assets/html/evolution/fontana_form.html

```diff
@@ -0,0 +1,47 @@
+<html lang="en">
+	<head>
+    	<meta charset="UTF-8">
+    	<meta name="viewport" content="width=device-width, initial-scale=1">
+    	<script src="https://cdnjs.cloudflare.com/ajax/libs/prefixfree/1.0.7/prefixfree.min.js"></script>
+  
+    	<link rel="stylesheet"
+          href="https://necolas.github.io/normalize.css/5.0.0/normalize.css">
+    	<link href="../../styles/main.css" rel="stylesheet">
+    	<link rel="stylesheet"
+    	       href="Roboto+Slab|Roboto:300,400,700" rel="stylesheet">
+    	<title>Form Fontana</title>
+	</head>    
+	<body>
+    <section class="evo_c-fontana-form">
+		<header id="fontanta_form__header">Form inspired by Italian artist/painter Lucio Fontana</header>
+    <section class="evo_c-fontana-form">
+		  <form id="evo_c-fontana-form"> 
+  		<input id="evo_c-fontana-form__first-name" class="evo_c-fontana-form__inputAttr" type="text" 
+             placeholder="FIRST NAME" minlength="3" maxlength="25" required> 
+  
+  		<input id="evo_c-fontana-form__last-name" class="evo_c-fontana-form__inputAttr" type="text" 
+            placeholder="LAST NAME" minLength="3" maxlength="25" required>
+
+  		<input id="evo_c-fontana-form__email"  class="evo_c-fontana-form__inputAttr type="
+             email"             
+             placeholder="EMAIL" pattern="[a-zA-Z0-9.-_]{1,}@[a-zA-Z.-]{2,}[.]{1}[
+                   a-zA-Z]{2,}" required>
+ 
+		 	<textarea id="evo_c-fontana-form__message" type="text" cols="45" rows="5" required 
+                placeholder="MESSAGE"></textarea>
+  
+  		<div id="evo_c-fontana-form__textbottom"></div>
```

Comment:
Hey @ModernTek 

Just a few tiny issues with the naming of your classes. 
* For example, `evo_c-fontana-form__button_white` would be `evo_c-fontana-form__button--light`. Notice that I have replaced `_white` with `--light`.  Imagine you changed your mind and no longer wanted an element to have a white line, rather a light shade of gray line, the class `vo_c-fontana-form__button_white__line` would no longer make much sense since the line is no longer of a white color.
* `evo_c-fontana-form__button_white__line` breaks BEM naming convention. It would rather be something like `evo_c-fontana-form__button--light-line`.

Lemme refer you to our [guide on naming classes](https://github.com/BovAcademy-opensource/evolution-ui/blob/development/CODE_GUIDE.md#bem-block-element-modifier)